src/responses/bullyisms.ts

```diff
@@ -12,4 +12,15 @@ export const bullyTemplate = new Response(
   BULLY_TEMPLATE_RESPONSE
 )
 
-export const bullyisms = [bullyTemplate]
+const MILLENIAL_RAGE = RegExp('(?:^|\\B)#(?![0-9_]+\\b)([a-zA-Z0-9_]{1,30})(\\b|\\r)') // https://stackoverflow.com/a/42551826
+const MILLENIAL_RAGE_RESPONSE = msg => {
+  return "These youngins and their instasnaps and their snapbooks. Back in my day we talked to people face to face. 😡"
```

Comment:
Outside of scope of the request but I think we may want some of these responses that are based on random stuff said in chat to have a probability of triggering. Maybe it would make OldMan more realistic if he responds with rage to a hashtag only 20% of the time.

 I'll add a separate feature request to add probability to the Response class and use it if set.

Reply:
That makes sense. Would probably want to add a constructor param in the Response class where a percentage can be passed, that defaults to 100 if not given.